openshift/system/Dockerfile

```diff
@@ -2,17 +2,24 @@ FROM quay.io/centos7/ruby-26-centos7
 
 USER root
 
+ENV RUBY_MAJOR_VERSION=2 \
+    RUBY_MINOR_VERSION=6
+
+ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}" \
+    RUBY_SCL_NAME_VERSION="${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
```

Comment:
Why do we need to ENV variables with the same value? Can't we just use `RUBY_VERSION`?

Reply:
They are not the same :upside_down_face: 
One has a dot (`2.6`), and other does not (`26`)